exercises/complex-numbers/canonical-data.json

```diff
@@ -0,0 +1,251 @@
+{
+  "exercise": "complex-numbers",
+  "version": "1.0.0",
+  "comments": [
+    " The canonical data assumes mathematically correct real   ",
+    " numbers. The testsuites should consider rounding errors  ",
+    " instead of testing for exact values for any non-integer  ",
+    " tests.                                                   ",
+    " Complex numbers z are represented as arrays [x, y] so    ",
+    " that z = x + i * y.                                      "
+  ],
+  "cases": [
+    {
+      "description": "Imaginary unit",
```

Comment:
I can move it under Multiplication but I think it should be seperate and on top because it's the most important property of complex numbers.